otter/controller.py

```diff
@@ -98,6 +98,16 @@ def __init__(self, tenant_id, group_id, operation, extra=None):
             fmt.format(t=tenant_id, g=group_id, o=operation))
 
 
+class TenantSuspendedError(Exception):
```

Comment:
Just a thought; you might want to have `tenant_id` as an actual attribute, in case anyone handling this error ever wants to inspect which tenant ID it is without regexing it out of the `str`.